addons/web_editor/controllers/main.py

```diff
@@ -278,7 +278,7 @@ def get_image_info(self, image_id=None, xml_id=None):
     def get_assets_editor_resources(self, key, get_views=True, get_scss=True, bundles=False, bundles_restriction=[]):
         # Related views must be fetched if the user wants the views and/or the style
         views = request.env["ir.ui.view"].get_related_views(key, bundles=bundles)
-        views = views.read(['name', 'id', 'key', 'xml_id', 'arch', 'active', 'inherit_id'])
+        views = views.read(['name', 'id', 'key', 'xml_id', 'arch', 'active', 'inherit_id', 'website_id'])
```

Comment:
Can't reference 'website_id' in web_editor ?

Reply:
:postal_horn: